### ISSUE TITLE
Overview detail information shows 'Something went wrong' when trying to display data (partially) before the first date of usage

### ISSUE THREAD

**Issue by CTHRU** [umami-software/umami#2752]
### Describe the Bug

The overview detail information shows 'Something went wrong (see picture below) when the date range is (partially) before the first date of usage. There's also an error on the terminal output. See screen below.

![image](https://github.com/umami-software/umami/assets/52134985/ff6a405f-0e20-4d70-8960-cd80cc20ad0e)

### Database

PostgreSQL

### Relevant log output

```shell
code: 'P2010',
umami     |   clientVersion: '5.12.1',
umami     |   meta: { code: '2201X', message: 'ERROR: OFFSET must not be negative' }
umami     | }
umami     | PrismaClientKnownRequestError: 
umami     | Invalid `prisma.$queryRawUnsafe()` invocation:
umami     | 
umami     | 
umami     | Raw query failed. Code: `2201X`. Message: `ERROR: OFFSET must not be negative`
umami     |     at In.handleRequestError (/app/node_modules/@prisma/client/runtime/library.js:122:6854)
umami     |     at In.handleAndLogRequestError (/app/node_modules/@prisma/client/runtime/library.js:122:6188)
umami     |     at In.request (/app/node_modules/@prisma/client/runtime/library.js:122:5896)
umami     |     at async l (/app/node_modules/@prisma/client/runtime/library.js:127:11167)
umami     |     at async m (/app/.next/server/pages/api/websites/[websiteId]/metrics.js:1:3217)
umami     |     at async K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.prod.js:20:16545)
umami     |     at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.prod.js:20:16981)
umami     |     at async NextNodeServer.runApi (/app/node_modules/next/dist/server/next-server.js:554:9)
umami     |     at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:266:37)
umami     |     at async NextNodeServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:791:17) {
umami     |   code: 'P2010',
umami     |   clientVersion: '5.12.1',
umami     |   meta: { code: '2201X', message: 'ERROR: OFFSET must not be negative' }
umami     | }
```

### Which Umami version are you using? (if relevant)

v2.11.3

### Which browser are you using? (if relevant)

Firefox

### How are you deploying your application? (if relevant)

Docker

**Comment by franciscao633** [comment]
We have this bug fixed in this [commit](https://github.com/umami-software/umami/commit/42b475b0619c35301d178822e6114da8065310c4). It will come out with the next release.

**Comment by mikecao** [comment]
Fixed in v2.12.0.
